src/pylon_camera/pylon_camera_parameter.cpp

```diff
@@ -204,6 +207,16 @@ void PylonCameraParameter::readFromRosParameterServer(const ros::NodeHandle& nh)
             }
         }
     }
+
+
+    
+    if ( nh.hasParam("whitebalance_auto") )
```

Comment:
@jonbinney:  
Wouldn't I want to see if the parameter is already set? Dynamic reconfigure should be setting it at startup, but what if the launchfile sets a different value?